src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -136,9 +148,15 @@ object EventMessages {
         SlackMessage(List(SlackBlock("section", SlackText("mrkdwn", "The Jenkins backup has failed. Please check the maintenance window in systems manager")))).some
       }
     }
+
+    override def sqs(incomingEvent: SSMMaintenanceEvent, context: Unit): Option[SqsMessageDetails] = Option.empty
   }
 
   implicit val exportStatusEventMessages: Messages[ExportStatusEvent, Unit] = new Messages[ExportStatusEvent, Unit] {
+    private def sendToTransformEngine(ev: ExportStatusEvent): Boolean = {
+      ev.success && ev.successDetails.isDefined && ev.successDetails.get.consignmentType == "judgment"
```

Comment:
This is my new irrational dislike of calling .get on Option so feel free to ignore but can this be simplified slightly like `ev.success && ev.successDetails.exists(_.consignmentType == "judgment")`